renderdoc/driver/vulkan/wrappers/vk_dynamic_funcs.cpp

```diff
@@ -24,6 +24,14 @@
 
 #include "../vk_core.h"
 
+#define ILLEGAL_EDS3_CALL(func)                                            \
+  RDCERR("Aborting the application due to illegal function call: " #func); \
+  abort();
```

Comment:
As a side note, this would be a use for `RDCFATAL` rather than `RDCERR` and `abort`. That said generally speaking I avoid fatal errors except for extreme scenarios where there is really no reasonable way to continue like memory allocation failures. In this case I'd rather print the error and continue on.